src/main/java/com/hubspot/jinjava/lib/filter/EscapeJinjavaFilter.java

```diff
@@ -47,8 +55,18 @@ public static String escapeJinjavaEntities(String input) {
     return StringUtils.replaceEach(input, TO_REPLACE, REPLACE_WITH);
   }
 
+  public static String escapeFullJinjavaEntities(String input) {
+    return input
+      .replace("{{", BLBRACE + BLBRACE)
```

Comment:
should we use https://github.com/HubSpot/jinjava/blob/master/src/main/java/com/hubspot/jinjava/tree/parse/TokenScannerSymbols.java here instead of hard-coding them?

Reply:
It gets a little tricky when using other characters and determining how they should be escaped so I specified in the description that it's only for the default: `default expression, tag, or comment marks`